syntax = "proto3";

service ChatService {
  rpc SendMessage (ChatMessage) returns (ChatResponse);
}

message ChatMessage {
  string message = 1;
  string user = 2;
}

message ChatResponse {
  string response = 1;
}
